### ISSUE TITLE
Number of markers in the cluster not showing

### ISSUE THREAD

**Issue by GeovanePiccinin** [googlemaps/v3-utility-library#549]
Just following the examples the clusters are created but the number of markers inside it are not being shown. Based on the examples wouldn't it be expected to be shown automatically like it is in the Marker Clusterer?

**Comment by jpoehnelt** [comment]
Assuming this is for markerclustererplus? What version? Can you make reproducible example on jsfiddle/codepen/etc?

**Comment by GeovanePiccinin** [comment]
@jpoehnelt 
Yes, this was for markerclusterplus. I just downloaded the most recent one from github. I was looking in the markerclusterer.js file and I think I found the problem.

In the method show() when the div is created it seems a plus sign is missing.

`
...
show() {
...
img += "'>";
      this.div_.innerHTML = img + "<div " +
        "aria-label='" + ariaLabel + "' "
      "tabindex='0' " +
...`

I just add a plus sign after the: ariaLabel + "' " **+**
(It is about line 271 seeing the raw js.)

now it looks like this and it is working:

`...
show() {
...
img += "'>";
      this.div_.innerHTML = img + "<div " +
        "aria-label='" + ariaLabel + "' " +
      "tabindex='0' " +
...`
